hydroflow/Cargo.toml

```diff
@@ -22,6 +22,10 @@ python = [ "dep:pyo3" ]
 name = "kvs_bench"
 required-features = [ "nightly" ]
 
+[[example]]
+name = "python_udf"
+required-features = [ "python" ]
```

Comment:
Do we also need to gate the dependency itself or is just not building pyo3 stuff enough?

Reply:
this PR still doesn't fix the m1 build AFAIK, just fixes some logical errors in the build

Reply:
e.g. before you could run `cargo run -p hydroflow --example python_udf --no-default-features`